tag/mutations_test.go

```diff
@@ -0,0 +1,82 @@
+package tag_test
+
+import (
+	"testing"
+
+	"github.com/jinzhu/gorm"
+	"github.com/traggo/server/schema"
+	"github.com/traggo/server/tag"
+	"github.com/traggo/server/test"
+)
+
+func Test_createTag_succeeds(t *testing.T) {
+	query := `
+mutation {
+   createTag(key: "new tag", color: "#123456", type:novalue) {
+      key,
+      color,
+      type
+   }
+}`
+	expected := `
+{
+  "createTag": {
+    "color": "#123456",
+    "key": "new tag",
+    "type": "novalue"
+  }
+}`
+	test.NewGQL(t).Mutations(tag.Mutations).Exec(query).Succeeds(expected)
+}
+
+func Test_createTag_withExistingTag_fails(t *testing.T) {
+	createTag := func(db *gorm.DB) {
+		db.Create(schema.TagDefinition{Key: "existing tag", Color: "#fff", Type: schema.TypeSingleValue})
+	}
+	query := `
+mutation {
+   createTag(key: "existing tag", color: "#123456", type:novalue) {
+      key
+   }
+}`
+	expectError := "tag with key 'existing tag' does already exist"
+
+	test.NewGQL(t).Mutations(tag.Mutations).BeforeExec(createTag).Exec(query).Errs(expectError)
```

Comment:
Interesting test approach!

Edit: LOL I thought this is coming from a test library and not from https://github.com/traggo/server/pull/3/files#diff-48f5fadaf165634605e6a7e9f8c2949c .

Reply:
In https://github.com/traggo/server/pull/3/files#diff-c8f7c60b36faa33b80e854e7de49b103R81 you didn't use an "extra" variable. Here you do. Is it just about the readability, because the line would be to long?
Maybe adjust `Test_removeTag_fails` also, so the file looks consistent.

Reply:
Yup, readability is one of the most important things for tests (: